com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeReferenceVolume.cs

```diff
@@ -286,18 +288,33 @@ public partial class ProbeReferenceVolume
     {
         const int kTemporaryDataLocChunkCount = 8;
 
-        [System.Serializable]
+        [Serializable]
         internal class Cell
         {
-            public int index;
             public Vector3Int position;
-            public List<Brick> bricks;
-            public Vector3[] probePositions;
-            public SphericalHarmonicsL2[] sh;
-            public float[] validity;
+            public int index;
             public int minSubdiv;
             public int indexChunkCount;
             public int shChunkCount;
+
+            public ProbeVolumeSHBands shBands;
+
+            public NativeArray<Brick> bricks { get; internal set; }
+
+            // shData is swizzled for direct runtime upload
+            //
+            // L0r0, L0g0, L0b0, L1r0
```

Comment:
It'd be nice to have a very small function to this PR that moves from a probe index to a SphericalHarmonicsL2 (undoing the swizzling) as I am sure it'll be useful for debug and potentially other use cases.

Reply:
These exist:
ProbeGIBaking.WriteToShaderCoeffsL0L1/WriteToShaderCoeffsL2/ReadFromShaderCoeffsL0L1/ReadFromShaderCoeffsL2
DilatedProbe.FromSphericalHarmonicsShaderConstants/ToSphericalHarmonicsShaderConstants